Assignment2/src/Components/Header/SortMenu.tsx

```diff
@@ -15,25 +14,15 @@ export default function SortMenu(props: SortMenuProps) {
                 {
                     key: '1',
                     label: (
-                        <span onClick={selectedSortType('NameAsc')}>Name -- Asc</span>
+                        <span onClick={() => selectedSortType('name')}>Name</span>
                     ),
                 },
                 {
                     key: '2',
                     label: (
-                        <span onClick={selectedSortType('NameDsc')}>Name -- Dsc</span>
+                        <span onClick={() => selectedSortType('phone')}>Phone</span>
                     ),
-                },
-                {
-                    key: '3',
-                    label: (
-                        <span onClick={selectedSortType('PhoneAsc')}>Phone -- Asc</span>
-                    ),
-                },
-                {
-                    key: '4',
-                    label: (<span onClick={selectedSortType('PhoneDsc')}>Phone -- Dsc</span>),
-                },
+                }
             ]}
         />
     );
```

Comment:
why SortMenuProp?

rename to SortMenuItems
should be Array of String
const SortMenuItems: string[] = ["name", "phone"]
and this should be reused around the app for sorting